src/common/utxobased/keymanager/bip69.ts

```diff
@@ -0,0 +1,54 @@
+// Previous transaction hashes (in reversed byte-order) are to be sorted in ascending order, lexicographically.
+// In the event of two matching transaction hashes, the respective previous output indices will be compared by their integer value, in ascending order.
```

Comment:
Can we make these comments multiline and include a link to bip69 for the masthead comment?

Reply:
Done. Also I noticed the comments were all over the place so I fixed them